list_files/afarina.list

```diff
@@ -1222,7 +1226,7 @@ OH OH65 US33_S US33_N
 OH OH65 OH110 OH295
 OH OH65 OH235 I-280(9)
 OH OH66 OH47 US24
-OH OH66AltTrkDef OH66_N OH66_S
+OH OH66TrkEDef OH66_N OH66_S

```

Comment:
Happened to notice this line will still be an error. The route name is `OH66TrkEde`.